node-template/src/service.rs

```diff
@@ -105,9 +105,9 @@ pub fn new_full<C: Send + Default + 'static>(config: Configuration<C, GenesisCon
 
 		let aura = aura::start_aura::<_, _, _, _, _, AuraPair, _, _, _>(
 			aura::SlotDuration::get_or_compute(&*client)?,
-			client.clone(),
-			select_chain,
 			client,
```

Comment:
the fact that it's so easy to get this wrong means that we should probably not implement `BlockImport` on `Client`.